tests/run-pass/concurrency/thread-spawn.stderr

```diff
@@ -0,0 +1,2 @@
+thread 'main' panicked at 'unsupported Miri functionality: Miri does not support concurrency on Windows', RUSTLIB/std/src/sys/windows/thread.rs:LL:COL
```

Comment:
This is modeled after https://github.com/rust-lang/miri/blob/28717a0e907ef427946a39adca1cbe3fa4b1e0f4/tests/run-pass/backtrace-api.stderr which I didn't realize has a special pragma in its `.rs` file for cleaning up stderr. Attempting to fix now.

Reply:
Possibly fixed? My toolchain somehow broke after making this change:

```rust
test [ui] run-pass\cfg_miri.rs ... FAILED
normalized stderr:
error: internal compiler error: compiler/rustc_metadata/src/rmeta/decoder.rs:1163:17: get_optimized_mir: missing MIR for `DefId(1:8771 ~ std[ab61]::rt::lang_start_internal)`

thread 'rustc' panicked at 'Box<Any>', compiler/rustc_errors/src/lib.rs:1007:9
stack backtrace:
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.

note: the compiler unexpectedly panicked. this is a bug.

note: we would appreciate a bug report: https://github.com/rust-lang/rust/issues/new?labels=C-bug%2C+I-ICE%2C+T-compiler&template=ice.md

note: rustc 1.54.0-nightly (da865095c 2021-06-02) running on x86_64-pc-windows-msvc

note: compiler flags: -C prefer-dynamic

```

RIP

Reply:
I am not sure what this test is intended to cover -- we already test the new panic elsewhere, I'd rather not have dozens of tests for each of the ways it can be triggered.^^

Also the test name indicates that this is about concurrency or thread spawning, when it really is more about panic-on-unsupported-concurrency.